docs/DeveloperGuide.md

```diff
@@ -86,6 +86,55 @@ Guarantees: Editing of internship application is successful
   1b1. InternHunter displays an error message and informs the user of the valid input format. </br>
   Use case resumes from step 1.
 
+### Use case: UC10 - Delete a skill or achievement
+
+#### MSS
+
+Precondition: User already has an existing list of skills and achievements. </br>
+Guarantees: Deletion of skill or achievement is successful
+
+1.  User requests to delete a skill or achievement from the user profile and provides the index.
+2.  InternHunter removes the skill or achievement from the user profile. </br>
+    Use case ends.
+
+#### Extensions
+
+1a. InternHunter detects that the index entered is out of bounds. </br>
```

Comment:
Do you think it is good to be overly specific about the type of error? If so then there are many more extensions to include such as inputting an index that is not a number for example.

Reply:
Yup we could keep it simple currently and once we merge them in the future, we could include those extensions.